android/jacoco.gradle.kts

```
@@ -34,9 +34,34 @@ tasks.create(name = "fhircoreJacocoReport", type = JacocoReport::class) {
       "**/*\$Result$*.*",
       // Data classes and enums without functions or methods
       "**/org/smartregister/fhircore/engine/auth/*",
-      "**/org/smartregister/fhircore/engine/configuration/*",
-      "**/org/smartregister/fhircore/engine/data/remote/model/*",
-      "**/org/smartregister/fhircore/engine/domain/model/*",
+      "**/org/smartregister/fhircore/engine/configuration/ConfigType*.*",
+      "**/org/smartregister/fhircore/engine/configuration/app/*",
+      "**/org/smartregister/fhircore/engine/configuration/geowidget/*",
+      "**/org/smartregister/fhircore/engine/configuration/navigation/*",
+      "**/org/smartregister/fhircore/engine/configuration/profile/*",
+      "**/org/smartregister/fhircore/engine/configuration/register/*",
+      "**/org/smartregister/fhircore/engine/configuration/report/measure/*",
+      "**/org/smartregister/fhircore/engine/configuration/view/CardViewProperties*.*",
+      "**/org/smartregister/fhircore/engine/configuration/view/ColumnProperties*.*",
+      "**/org/smartregister/fhircore/engine/configuration/view/CompoundTextProperties*.*",
+      "**/org/smartregister/fhircore/engine/configuration/view/ListProperties&*.*",
```

is the ampersand intentional?
```suggestion
      "**/org/smartregister/fhircore/engine/configuration/view/ListProperties*.*",
```